src/Material.h

```diff
@@ -153,29 +193,30 @@ class Material {
   void setSigmaF(double* xs, int num_groups);
   void setNuSigmaF(double* xs, int num_groups);
   void setChi(double* xs, int num_groups);
-  void setBuckling(double* xs, int num_groups);
   void setDifCoef(double* xs, int num_groups);
   void setDifHat(double* xs, int num_groups);
   void setDifTilde(double* xs, int num_groups);
+  void setBuckling(double* xs, int num_groups);
 
   void setSigmaTByGroup(double xs, int group);
   void setSigmaAByGroup(double xs, int group);
   void setSigmaFByGroup(double xs, int group);
   void setNuSigmaFByGroup(double xs, int group);
   void setSigmaSByGroup(double xs, int origin, int destination);
+  void setSigmaSByGroupInline(double xs, int origin, int destination);
```

Comment:
As I mentioned before, I'm not sure how relevant an inline setter routine is here.